middleware/browse/browse_test.go

```diff
@@ -1,14 +1,17 @@
 package browse
 
 import (
+	//"bytes"
```

Comment:
There seem to be a few extra, commented lines that snuck in - maybe take those out when you have a chance. There's a few more below.